src/test/java/nextstep/subway/line/LineAcceptanceTest.java

```diff
@@ -23,15 +31,28 @@
 	public static final String COLOR2 = "bg-blue-600";
 	public static final String NAME2 = "구분당선";
 	public static final String LOCATION_HEADER_NAME = "Location";
+	public static final Station STATION1 = new Station("삼성역");
```

Comment:
```suggestion
	public static final Station 삼성역 = new Station("삼성역");
```
이런식으로 한글 변수명을 사용해도 테스트 가독성에 도움이 될 것 같습니다 :)